src/components/Track.tsx

```diff
@@ -133,12 +133,14 @@ const TrackConsumer: React.FC<TrackConsumerProps> = ({
           );
 
           if (!isEqual) {
+            sequencer.current.stop();
```

Comment:
Thanks for raising this PR. Just checking, will this stop the sequence if you change the steps while playing?

FYI - in the upcoming `tone-14` branch, I had to rewrite how the sequencer works as Tone no longer has `add` and `remove` in the `Sequence` (for some reason). 

Perhaps this may fix this issue? I'll try and get a canary release out if you'd like to try it.
**https://github.com/unkleho/reactronica/blob/tone-14/src/components/Track.tsx#L156-L165**

Reply:
Ah, yeah, it does stop it while playing. For my use case, I think this is OK, but I could see that being a problem for users who are more actively 'live-editing' or performing with an app that's built around these components. 

The effect for changing steps in the tone-14 branch looks much simpler though 👍  I'd definitely try it out if you have some time to push out a pre-release version.